src/piton/labelers/core.py

```diff
@@ -471,7 +471,7 @@ def label(self, patient: Patient) -> List[Label]:
         last_time = None
         for time in prediction_times:
             if last_time is not None:
-                assert time > last_time, f"Must be ascending prediction times, instead got {last_time} <= {time}"
+                assert time >= last_time, f"Must be ascending prediction times, instead got {last_time} <= {time}"
```

Comment:
This is incorrect. It needs to be > in order for the rest of the logic to work correctly.